test/regression-evm-semantics-39dd1e5/test-add0-tmp.in.GYhmR347rd

```diff
@@ -0,0 +1 @@
(inj{SortString{}, SortJSONKey{}}(\dv{SortString{}}("gas")),inj{SortString{}, SortJSON{}}(\dv{SortString{}}("0x013874"))),LblJSONs{}(LblJSONEntry{}(inj{SortString{}, SortJSONKey{}}(\dv{SortString{}}("logs")),inj{SortString{}, SortJSON{}}(\dv{SortString{}}("0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347"))),LblJSONs{}(LblJSONEntry{}(inj{SortString{}, SortJSONKey{}}(\dv{SortString{}}("out")),inj{SortString{}, SortJSON{}}(\dv{SortString{}}("0x"))),LblJSONs{}(LblJSONEntry{}(inj{SortString{}, SortJSONKey{}}(\dv{SortString{}}("post")),LblJSONObject{}(LblJSONs{}(LblJSONEntry{}(inj{SortString{}, SortJSONKey{}}(\dv{SortString{}}("0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6")),LblJSONObject{}(LblJSONs{}(LblJSONEntry{}(inj{SortString{}, SortJSONKey{}}(\dv{SortString{}}("balance")),inj{SortString{}, SortJSON{}}(\dv{SortString{}}("0x0de0b6b3a7640000"))),LblJSONs{}(LblJSONEntry{}(inj{SortString{}, SortJSONKey{}}(\dv{SortString{}}("code")),inj{SortString{}, SortJSON{}}(\dv{SortString{}}("0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff01600055"))),LblJSONs{}(LblJSONEntry{}(inj{SortString{}, SortJSONKey{}}(\dv{SortString{}}("nonce")),inj{SortString{}, SortJSON{}}(\dv{SortString{}}("0x00"))),LblJSONs{}(LblJSONEntry{}(inj{SortString{}, SortJSONKey{}}(\dv{SortString{}}("storage")),LblJSONObject{}(LblJSONs{}(LblJSONEntry{}(inj{SortString{}, SortJSONKey{}}(\dv{SortString{}}("0x00")),inj{SortString{}, SortJSON{}}(\dv{SortString{}}("0xfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe"))),Lbl'Stop'List'LBraQuot'JSONs'QuotRBraUnds'JSONs{}()))),Lbl'Stop'List'LBraQuot'JSONs'QuotRBraUnds'JSONs{}())))))),Lbl'Stop'List'LBraQuot'JSONs'QuotRBraUnds'JSONs{}()))),LblJSONs{}(LblJSONEntry{}(inj{SortString{}, SortJSONKey{}}(\dv{SortString{}}("pre")),LblJSONObject{}(LblJSONs{}(LblJSONEntry{}(inj{SortString{}, SortJSONKey{}}(\dv{SortString{}}("0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6")),LblJSONObject{}(LblJSONs{}(LblJSONEntry{}(inj{SortString{}, SortJSONKey{}}(\dv{SortString{}}("balance")),inj{SortString{}, SortJSON{}}(\dv{SortString{}}("0x0de0b6b3a7640000"))),LblJSONs{}(LblJSONEntry{}(inj{SortString{}, SortJSONKey{}}(\dv{SortString{}}("code")),inj{SortString{}, SortJSON{}}(\dv{SortString{}}("0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff01600055"))),LblJSONs{}(LblJSONEntry{}(inj{SortString{}, SortJSONKey{}}(\dv{SortString{}}("nonce")),inj{SortString{}, SortJSON{}}(\dv{SortString{}}("0x00"))),LblJSONs{}(LblJSONEntry{}(inj{SortString{}, SortJSONKey{}}(\dv{SortString{}}("storage")),LblJSONObject{}(Lbl'Stop'List'LBraQuot'JSONs'QuotRBraUnds'JSONs{}())),Lbl'Stop'List'LBraQuot'JSONs'QuotRBraUnds'JSONs{}())))))),Lbl'Stop'List'LBraQuot'JSONs'QuotRBraUnds'JSONs{}()))),Lbl'Stop'List'LBraQuot'JSONs'QuotRBraUnds'JSONs{}()))))))))))),Lbl'Stop'List'LBraQuot'JSONs'QuotRBraUnds'JSONs{}())))))))
```

Comment:
Is there an explanation for why some of these have file extensions other than `.kore`?

Reply:
That is due to the frontend changing the naming of the files in the temporary run directory. There's a discussion about it in our #k channel if you're interested.